src/tests/ftest/util/pool_security_test_base.py

```diff
@@ -344,7 +344,8 @@ def verify_pool_readwrite(self, uuid, action, expect='Pass'):
         daos_cmd = DaosCommand(self.bin)
         daos_cmd.exit_status_exception = False
         if action.lower() == "write":
-            result = daos_cmd.container_create(pool=uuid)
+            # pylint: disable=method-hidden
+            result = daos_cmd._get_result(("container", "create"), pool=uuid)
```

Comment:
(pylint-protected-access) Access to a protected member _get_result of a client class